tests/main/snap-service/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Check that `snap run --command=reload` works
+
+kill-timeout: 1m
+execute: |
+  echo "When the service snap is installed"
```

Comment:
We've discussed standardizing both snap.yaml and task.yaml on four spaces for all indents. Not sure where we stand on that by now.